docs/source/api/v4/cdn_notifications.rst

```diff
@@ -0,0 +1,201 @@
+:notification:	The content of the notification
+:user:			The user responsible for creating the notification
+
+.. code-block:: http
+	:caption: Response Example
+
+	HTTP/1.1 200 OK
+	Access-Control-Allow-Credentials: true
+	Access-Control-Allow-Headers: Origin, X-Requested-With, Content-Type, Accept, Set-Cookie, Cookie
+	Access-Control-Allow-Methods: POST,GET,OPTIONS,PUT,DELETE
+	Access-Control-Allow-Origin: *
+	Content-Encoding: gzip
+	Content-Type: application/json
+	Set-Cookie: mojolicious=...; Path=/; Expires=Mon, 02 Dec 2019 22:51:14 GMT; Max-Age=3600; HttpOnly
+	Whole-Content-Sha512: F2NmDbTpXqrIQDX7IBKH9+1drtTL4XedSfJv6klMgLEZwbLCkddIXuSLpmgVCID6kTVqy3fTKjZS3U+HJ3YUEQ==
+	X-Server-Name: traffic_ops_golang/
+	Date: Mon, 02 Dec 2019 21:51:14 GMT
+	Content-Length: 128
+
+	{ "response": [
+		{
+			"cdn": "cdn1",
+			"lastUpdated": "2019-12-02 21:49:08+00",
+			"notification": "the content of the notification",
+			"user": "username123",
+		}
+	]}
+
+``POST``
+========
+Creates a notification for a specific CDN.
+
+.. note:: Currently only one notification per CDN is supported.
+
+:Auth. Required:	Yes
+:Roles Required:	"admin" or "operations"
+:Response Type:		Object
+
+Request Structure
+-----------------
+:cdn:			The name of the CDN to which the notification shall belong
+:notification:	The content of the notification
+
+.. code-block:: http
+	:caption: Request Example
+
+	POST /api/4.0/cdn_notifications HTTP/1.1
+	User-Agent: python-requests/2.22.0
+	Accept-Encoding: gzip, deflate
+	Accept: */*
+	Connection: keep-alive
+	Cookie: mojolicious=...
+	Content-Length: 29
+
+	{"cdn": "cdn1", "notification": "the content of the notification"}
+
+
+Response Structure
+------------------
+:cdn:			The name of the CDN to which the notification belongs to
+:lastUpdated:	The time and date this server entry was last updated in an ISO-like format
+:notification:	The content of the notification
+:user:			The user responsible for creating the notification
+
+
+.. code-block:: http
+	:caption: Response Example
+
+	HTTP/1.1 200 OK
+	Access-Control-Allow-Credentials: true
+	Access-Control-Allow-Headers: Origin, X-Requested-With, Content-Type, Accept, Set-Cookie, Cookie
+	Access-Control-Allow-Methods: POST,GET,OPTIONS,PUT,DELETE
+	Access-Control-Allow-Origin: *
+	Content-Encoding: gzip
+	Content-Type: application/json
+	Set-Cookie: mojolicious=...; Path=/; Expires=Mon, 02 Dec 2019 22:49:08 GMT; Max-Age=3600; HttpOnly
+	Whole-Content-Sha512: mx8b2GTYojz4QtMxXCMoQyZogCB504vs0yv6WGly4dwM81W3XiejWNuUwchRBYYi8QHaWsMZ3DaiGGfQi/8Giw==
+	X-Server-Name: traffic_ops_golang/
+	Date: Mon, 02 Dec 2019 21:49:08 GMT
+	Content-Length: 150
+
+	{ "alerts": [
+		{
+			"text": "notification was created.",
+			"level": "success"
+		}
+	],
+	"response": {
+		"cdn": "cdn1",
+		"lastUpdated": "2019-12-02 21:49:08+00",
+		"notification": "the content of the notification",
+		"user": "username123",
+	}
+}
```

Comment:
This closing brace needs to be indented. Or probably just at the end of the previous line.